lib/competitions/data/users.yml

```diff
@@ -1,81 +1,68 @@
 user1:
   first_name: 'John'
   last_name: 'Doe'
-  email: 'j-doe@northwestern.edu'
+  email: 'j-doe@x.edu'
+  upn: 'jd1@x.edu'
   system_admin: true
-  password: 'secret'
   grant_creator: false
 user2:
   first_name: 'Exene'
   last_name: 'Cervenka'
-  email: 'e-cervenaka@northwestern.edu'
+  email: 'e-cervenaka@x.edu'
+  upn: 'ec1@x.edu'
   system_admin: false
-  password: 'secret'
   grant_creator: false
   era_commons: 'ECervenk90069'
+  upn: 'ec1@x.edu'
 user3:
   first_name: 'Billy'
   last_name: 'Zoom'
-  email: 'b-zoom@northwestern.edu'
+  email: 'b-zoom@x.edu'
+  upn: 'bz1@x.edu'
   system_admin: false
-  password: 'secret'
   grant_creator: false
 user4:
   first_name: 'D.J.'
   last_name: 'Bonebrake'
-  email: 'd-bonebrake@northwestern.edu'
+  email: 'd-bonebrake@x.edu'
+  upn: db1@x.edu
   system_admin: false
-  password: 'secret'
   grant_creator: false
   era_commons: 'DBonebr90069'
 user5:
```

Comment:
Do we need this second group of test users? They were added to test org, as I recall.

Reply:
I think that is true. Does it hurt to keep them in?